[2025-04-01T20:19:05.347+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-01T20:19:05.389+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-01T20:19:00.785419+00:00 [queued]>
[2025-04-01T20:19:05.399+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-01T20:19:00.785419+00:00 [queued]>
[2025-04-01T20:19:05.400+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-01T20:19:05.416+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-01 20:19:00.785419+00:00
[2025-04-01T20:19:05.424+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=326) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T20:19:05.425+0000] {standard_task_runner.py:63} INFO - Started process 350 to run task
[2025-04-01T20:19:05.425+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'manual__2025-04-01T20:19:00.785419+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpps2o0ies']
[2025-04-01T20:19:05.426+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask upload_to_minio
[2025-04-01T20:19:05.447+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-01T20:19:05.486+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-01T20:19:00.785419+00:00 [running]> on host 32b02857b00b
[2025-04-01T20:19:05.582+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T20:19:00.785419+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T20:19:00.785419+00:00'
[2025-04-01T20:19:05.583+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-01T20:19:05.596+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-01T20:19:05.597+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-01T20:19:05.599+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-01T20:19:05.613+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=manual__2025-04-01T20:19:00.785419+00:00, execution_date=20250401T201900, start_date=20250401T201905, end_date=20250401T201905
[2025-04-01T20:19:05.640+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-01T20:19:05.661+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-01T20:19:05.663+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
